src/broker/log.c

```diff
@@ -244,54 +243,40 @@ static int logbuf_set_filename (logbuf_t *logbuf, const char *destination)
     return 0;
 }
 
+static const char *int_to_string (int n)
+{
+    static char s[32];
+    (void)snprintf (s, sizeof (s), "%d", n);
```

Comment:
Someone (not me!) will probably want a comment that `%d` of an `int` can never require more than 32 bytes (actually I think it is only 12 or 13 (for 4 byte int) but it doesn't hurt to have more than double)